### PR TITLE
Release 6.6.0

### PR DESCRIPTION
**Pull request by katspaugh**
6.6.0 (12.03.2023)
------------------
- [Regions] Fix undefined content element on remove (#2713)
- Zoom optimisation for Waves and matching implementation for Spectrograms (#2646)
- Fix: release workflow permissions (#2709)

### CODE REVIEW

**Comment by katspaugh** [comment]
I had to publish 6.5.2 in the meanwhile because I had accidentally published an alpha version w/o the corresponding npm tag. Now it's back to normal.

**Comment by thijstriemstra** [comment]
> I had to publish 6.5.2 in the meanwhile because I had accidentally published an alpha version w/o the corresponding npm tag. Now it's back to normal.

I have the feeling you're rushing a lot of things (merges, reviews, releases); why? These things would've been caught with simple reviews, right?

**Comment by katspaugh** [comment]
Yes, I definitely rushed things. Partly because I only have limited time on the weekend, and want to get as much things done as possible.
I mistakenly ran `npm publish` instead of `npm publish --tag alpha`.
Anyway, not an excuse to bypass reviews.

**Review by thijstriemstra — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 wavesurfer.js changelog
 =======================
 
+6.6.0 (12.03.2023)
+------------------
+- [Regions] Fix undefined content element on remove (#2713)
```
Comment (thijstriemstra) [inline CHANGES.md:6]: [Regions] should be  in a separate list (see other changelogs below for example)

**Review by thijstriemstra — CHANGES_REQUESTED** [review]

**Review by thijstriemstra — APPROVED** [review]

**Comment by katspaugh** [comment]
Looks like I broke the npm publish part of the deployment action.
https://github.com/wavesurfer-js/wavesurfer.js/actions/runs/4419688300/jobs/7748430370
Will publish manually at this time, and fix the action later.

**Comment by katspaugh** [comment]
Created an issue: #2723.

**Pull request merged**